### PR TITLE
[rfw] Restore RFW to 100% coverage after `ButtonBar` update

### PR DESCRIPTION
**Pull request by TahaTesser**
fixes [[`rfw`]  Restore RFW to 100% coverage after ButtonBar update](https://github.com/flutter/flutter/issues/142602)

### Before

BEFORE

### After 

AFTER

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I [linked to at least one issue that this PR fixes] in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[linked to at least one issue that this PR fixes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

File: packages/rfw/lib/src/flutter/material_widgets.dart [hunk 1]
```diff
@@ -140,18 +140,18 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
...
   },
 
-  // The ButtonBar widget in package:flutter/material.dart is planned to be deprecated
-  // in favor of the OverflowBar widget. This ButtonBar implementation uses the
-  // OverflowBar widget internally for backward compatibility. The ButtonBar
-  // widget in rfw package is not deprecated and will continue to be supported.
+  // The [ButtonBar] widget in Flutter's material library is planned to be deprecated
+  // in favor of the [OverflowBar] widget. This [ButtonBar] implementation uses the
+  // [OverflowBar] widget internally for backward compatibility. The [ButtonBar]
+  // widget in `rfw` package is not deprecated and will continue to be supported.
```
Comment (TahaTesser) [inline packages/rfw/lib/src/flutter/material_widgets.dart:146]: Updated this section to match for markdown format issues:

https://github.com/flutter/packages/blob/5b48c446976fd5f61316d58ab3fcc9888423c2e6/packages/rfw/lib/src/flutter/material_widgets.dart#L57-L68

File: packages/rfw/test/material_widgets_test.dart [hunk 2]
```diff
@@ -225,6 +227,82 @@ void main() {
     );
   });
 
+  testWidgets('Implement ButtonBar properties', (WidgetTester tester) async {
```
Comment (TahaTesser) [inline packages/rfw/test/material_widgets_test.dart:230]: This tests get the 100% coverage.

File: packages/rfw/lib/src/flutter/material_widgets.dart [hunk 3]
```diff
@@ -140,18 +140,18 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
...
   // The following parameters are no longer supported:
-  //  - buttonMinWidth
-  //  - buttonHeight
-  //  - buttonAlignedDropdown
+  //  - `buttonMinWidth`
+  //  - `buttonHeight`
+  //  - `buttonAlignedDropdown`
   //
-  // It is recommended to use the OverflowBar widget.
+  // It is recommended to use the [OverflowBar] widget.
```
Comment (Hixie) [inline packages/rfw/lib/src/flutter/material_widgets.dart:154]: this is a comment so it doesn't matter much either way, fwiw. But this is good too.

**Review by Hixie — APPROVED** [review]
LGTM 

thank you!!

**Pull request merged**